client/lib/signup/site-type.js

```diff
@@ -36,8 +39,8 @@ const getSiteTypePropertyDefaults = ( propertyKey ) =>
 			domainsStepSubheader: i18n.translate(
 				'Enter a keyword that describes your site to get started.'
 			),
```

Comment:
A header will not be displayed in a non sub header flow, can we remove the old site attributes. ?